invenio_records_permissions/policies/base.py

```diff
@@ -97,8 +97,7 @@ def excludes(self):
         excludes = [
             generator.excludes(**self.over) for generator in self.generators
         ]
-        excludes = set(chain.from_iterable(excludes))
-        # self.explicit_excludes |= excludes  # See invenio_access issue
+        self.explicit_excludes |= set(chain.from_iterable(excludes))
```

Comment:
Ultra minor: the set creation step is divided in two in the needs:

``` python

excludes = set(chain.from_iterable(excludes))

self.explicit_excludes |= excludes
```

But not needed at all :)

Reply:
:heavy_check_mark: